lib/satis/menus/menu.rb

```diff
@@ -3,20 +3,21 @@
 module Satis
   module Menus
     class Menu
-      attr_reader :items, :level
-
+      attr_reader :items, :level, :event
       def initialize(*args, **kwargs)
         @options = args.extract_options!
         @items = []
         @scope = Array.wrap(args.first)
         @level = kwargs[:level] || 0
+        @event = "mouseover->satis-menu#show mouseleave->satis-menu#hide"
```

Comment:
Can we make options[:event] either be `mouseover` or `click` perhaps and based on that set @event?

Reply:
This one I have set as default and dynamic one I have handled at line 20 below, please check if its okay
Thank you!

Reply:
Well I meant if a developer would set event to "mouseover" we make the view use "mouseover->satis-menu#show mouseleave->satis-menu#hide", and for click the same. Now you do something similar with @scope.include(:filter_menu).

Reply:
Fixed! Thank you.